app/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 26
```

Comment:
Do we need to set 26 as the sdk versions?

Reply:
I had imported another dependency before that triggered a warning, but that's gone now